Upgraded to Ember 2.x

sivakumar-kailasam opened the pull request:
Hi @jimmay5469 @elwayman02, I was just going to start on a side project which needs Github issues and I didn't want to write adapters from scratch. I saw that you folks were trying to get this up to speed on 2.x libs. So I've ported them. It has passing tests and I'm planning to raise PRs to introduce issues and labels but the upgrade is my first step towards that. Let me know if there's anything more required in this PR. Cheers 😄 

Implements #9

sivakumar-kailasam commented:
Fixes #10

elwayman02 commented on bower.json:
```diff
@@ -1,20 +1,16 @@
+    "ember-cli-test-loader": "0.2.2",
+    "ember-qunit": "0.4.20",
+    "ember-qunit-notifications": "0.1.0",
+    "jquery": "^2.2.3",
```
This can be removed, gets included by ember now

elwayman02 commented on bower.json:
```diff
@@ -1,20 +1,16 @@
+    "ember-qunit": "0.4.20",
+    "ember-qunit-notifications": "0.1.0",
+    "jquery": "^2.2.3",
+    "qunit": "~1.23.0"
```
I believe this can also be removed, should get included by ember-qunit

elwayman02 commented on bower.json:
```diff
@@ -1,20 +1,16 @@
+    "ember": "2.4.4",
+    "ember-cli-shims": "0.1.1",
+    "ember-cli-test-loader": "0.2.2",
+    "ember-qunit": "0.4.20",
```
ember-qunit is no longer necessary, can be removed I believe

elwayman02 commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-data-github",
-  "version": "0.0.6",
+  "version": "1.0.0",
```
Please revert. I'll update the package version via `ember-cli-release` when the new version is ready to go out. Since we're in a beta version still, this will probably be a 0.1.0 release rather than 1.0.0, but either way I'll make that change in a separate commit.

elwayman02 commented on package.json:
```diff
@@ -18,20 +18,26 @@
-    "broccoli-asset-rev": "^2.0.2",
-    "ember-cli": "0.2.3",
-    "ember-cli-app-version": "0.3.3",
-    "ember-cli-content-security-policy": "0.4.0",
```
Revert, this gets removed by the default blueprint now, but we should keep it, along with a proper CSP config for the dummy app.

elwayman02 commented on package.json:
```diff
@@ -18,20 +18,26 @@
+    "ember-load-initializers": "^0.5.1",
+    "ember-resolver": "^2.0.3",
+    "ember-try": "0.2.2",
+    "ember-welcome-page": "^1.0.1",
```
This is not necessary if we have our own dummy app implemented.

elwayman02 commented:
@sivakumar-kailasam this looks awesome! This repo has been third or so on my list to deal with, but I hadn't gotten to it yet. I really appreciate all the work you've done here, it looks great. I left a few minor comments on the PR, but once we address those I think this is good to go. Thanks!

sivakumar-kailasam commented:
@elwayman02 I was feeling *twitch*y 😉 about updating the version no too. I've addressed the feedback from the comments in the latest commit. Looking forward to raising further PRs so that I can finally start using it on my project 😄

elwayman02 commented:
I just headed to bed, but I'll look at this again in the morning and get it
merged if we're good! Thanks again. :)

On Sun, Apr 10, 2016, 2:49 AM Sivakumar Kailasam <notifications@github.com>
wrote:

> @elwayman02 <https://github.com/elwayman02> I was feeling *twitch*y 😉
> about updating the version no too. I've addressed the feedback from the
> comments in the latest commit. Looking forward to raising further PRs so
> that I can finally start using it on my project 😄
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/elwayman02/ember-data-github/pull/10#issuecomment-207951403>
>

sivakumar-kailasam commented:
Sure, wfm! Good night :)
